.travis.yml

```diff
@@ -47,10 +47,11 @@ before_install:
   - echo "[buildout]" > $HOME/.buildout/default.cfg
   - echo "download-cache = $HOME/buildout-cache/downloads" >> $HOME/.buildout/default.cfg
   - echo "eggs-directory = $HOME/buildout-cache/eggs" >> $HOME/.buildout/default.cfg
-  - pip install -r requirements.txt
+  - virtualenv -p `which python` .
```

Comment:
@wesleybl I try not to rely on virtualenv on Travis because this has been caused lots of trouble in the past.

Reply:
@tisto  Well for me it was the opposite. I've seen problems using Travis' Python. Travis' Python is not "clean". It comes with pre-installed packages, which can conflict with versions required in the buildout. In this case, virtualenv 16.1.0 is installed on Python 2, conflicting with the requirement `virtualenv>=20.0.35`. With this new requirement, all packages that don't use virtualenv in Travis will become a problem in Python 2.

I can uninstall the virtualenv of Travis's Python, but this will make the Travis configuration more complex. I don't know what problem you had with virtualenv but I recommend using it. For now, it's working.

Reply:
@tisto Well for me it was the opposite. I've seen problems using Travis' Python. Travis' Python is not "clean". It comes with pre-installed packages, which can conflict with versions required in the buildout. In this case, virtualenv 16.1.0 is installed on Python 2, conflicting with the requirement virtualenv>=20.0.35. With this new requirement, all packages that don't use virtualenv in Travis will become a problem in Python 2.

I can uninstall the virtualenv of Travis's Python, but this will make the Travis configuration more complex. I don't know what problem you had with virtualenv but I recommend using it. For now, it's working.

Reply:
I actively maintain > 30 Plone packages that use the kitconcept buildout. I simply can not maintain all those Travis configs individually. I will consider changing things when I run into problems with the current approach. Though we might move away from Travis anyways and right now I don't see the value in changing this back to using virtualenv on Travis without a good cause.